applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/naca0012_test_adjoint_potential_probe1_ref.dat

```diff
@@ -0,0 +1,3 @@
+# Results for "element" with Id # 6 at position: x: 0.49961; y: 0.09728; z: 0
+# time ADJOINT_VELOCITY_POTENTIAL
+0.0 -0.1760110243922251
```

Comment:
Same again

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668